docs/DeveloperGuide.adoc

```diff
@@ -978,6 +1007,46 @@ Use case resumes at step 2.
 +
 Use case resumes at step 6.
 
+[discrete]
+=== Use case: Add a remark
```

Comment:
A for effort, but actually we don't need to update this section (I think) haha

Reply:
Oh is it, do I need to remove it?